src/content/eol/2024/08/eol-08-14-24.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Change to user and license keys"
+summary: "On September 25, 2024, we're removing the ability to copy user and license API keys from the UI."
+publishDate: '2024-08-01'
+eolEffectiveDate: '2024-09-09'
+---
+
+Beginning September 25, 2024, we are removing the ability to copy API keys from [the UI](https://get.newrelic.com/MzQxLVhLUC0zMTAAAAGUiZ8sq6uwzvQMEjtdfLs43rOhHFRYwkHxXjt8wYs2qCJP2IkoVpEZiGhCVneo-hfy1Ucijgo=). The “copy key” option in the drop-down seen below will be removed. This applies to user and license keys, both new and existing. If you have an automation that is dependent on scraping keys from the UI, please integrate using the NerdGraph API.
```

Comment:
we are > we're

Should copy key be bolded and/or capitalized since it's a UI element it sounds like?

that is > that's

Should there be a link to the Nerdgraph API doc that tells people how to integrate?